server/src/test/java/org/opensearch/cluster/coordination/FollowersCheckerTests.java

```diff
@@ -787,4 +795,47 @@ public TransportResponse.Empty read(StreamInput in) {
         }
 
     }
+
+    public void testFollowerCheckTimeoutValue() {
+        Setting<TimeValue> setting1 = FOLLOWER_CHECK_TIMEOUT_SETTING;
+        Settings timeSettings1 = Settings.builder().put(setting1.getKey(), "60s").build();
+
+        try {
+            ClusterUpdateSettingsResponse response = client().admin()
+                .cluster()
+                .prepareUpdateSettings()
+                .setPersistentSettings(timeSettings1)
+                .execute()
+                .actionGet();
+
+            assertAcked(response);
+            assertEquals(timeValueSeconds(60), setting1.get(response.getPersistentSettings()));
```

Comment:
nit : Before updating the timeout value, we should assert the value we expect to ensure it is indeed different compared to what we are setting

Reply:
Although I am not sure if we need to validate the default value, I still gave it a try. I am unable to figure out a way to assert the previous value within `OpenSearchSingleNodeTestCase`. And, I am not sure if we should start using `OpenSearchIntegTestCase` just for this, given how much more expensive it is. What do you think?